hack/test/day-two/ceph-values.yaml

```
@@ -0,0 +1,8 @@
+cephClusterSpec:
+  resources:
+    mgr:
+      limits: {}
+      requests: {}
+    osd:
+      limits: {}
+      requests: {}
```

It probably should be like this:

```suggestion
    mgr: {}
    osd: {}
```

---

Unfortunately, the Rook operator threw some errors when the CephCluster CRD was configured that way. I'll try and dig out the logs for you.